docs/core/extensions/snippets/logging/getting-started-open-telemetry/getting-started-open-telemetry.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <RootNamespace>getting_started_open_telemetry</RootNamespace>
```

Comment:
Is this needed? (other projects don't have it)

Reply:
No it isn't needed, I just forgot to remove it.